core/operations/management/query_index_drop.cxx

```diff
@@ -29,31 +29,27 @@ query_index_drop_request::encode_to(encoded_request_type& encoded, http_context&
         return errc::common::invalid_argument;
     }
     encoded.headers["content-type"] = "application/json";
-    std::string keyspace = fmt::format("`{}`", bucket_name);
-    std::string query_context = keyspace;
+    std::string query_context = fmt::format("default:{}", bucket_name);
```

Comment:
Would it be better to use the namespace_id here?  Same as _query_index_create_?

Reply:
ooh good catch - I'll fix it and make sure I didn't do that elsewhere.   Someday, we may have other namespaces.

Reply:
Also those do exist -- the _default strings, somewhere... I'll look.